cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/update/ChangelogUpdateTest.java

```diff
@@ -104,6 +107,32 @@ public void testCloneVariantVariantSpecificChanges() {
         assertTrue(changesVariant1b.size() == changesVariant1.size());
     }
 
+    @Test
+    public void testChangesLogRightOrder() {
+        String variant1 = network.getVariantManager().getWorkingVariantId();
+        // Make changes, on Variant and base.
+        makeNetworkChangesVariantSpecific(network);
+        makeRemoveChanges(network);
+        List<IidmChange> changes =  new ArrayList<>(changelog.getChangesForVariant(variant1));
+        assertTrue(changes.size() > 1);
```

Comment:
Do you wan simply to check if the list is empty?

Reply:
To test the right ordering, the list of changes should have multiple items.